pyproject.toml

```diff
@@ -52,7 +52,7 @@ dependencies = [
     "pyarrow",
     "ray[default,data]>=2.49",
     "torch",
-    "transformers>=4.48.0",
+    "transformers==4.53.1",
```

Comment:
@sarahyurick Do you prefer pinning to this specific version here or just anything >=4.48 and < latest_release and leave the exact pin for internvideo?

No strong opinions.

Reply:
I have no preference, just want to ensure the GPU classifier tests pass.